### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by itsrlyAria**
Adding information on how OEMs can expedite apps during OOBE

### CODE REVIEW

File: desktop-src/updateorchestrator/index.md [hunk 1]
```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
 
 Our aim is to improve the functionality of this API and reduce impact from multiple automatic software updaters on Windows. We would appreciate your input through this [**brief survey**](https://aka.ms/UOAPISurvey) to help us understand how UpdateOrchestrator API can better serve your developer needs.
 
+## For OEMs wanting to Expedite Applications 
+
+>[!IMPORTANT]
+> Expediting Applications is only available for OEMs and only applicable in select regions. Expediting an application may have negative impacts to the out of box experience for new devices. This functionality is only available on select client builds.
+
+OEMs can register an update to be expedited by writing a new key in the existing staging folder for expedited apps: HKLM\Software\Microsoft\WindowsUpdate\Orchestrator\UScheduler_OOBE
```
Comment (gerardoc-ms) [inline desktop-src/updateorchestrator/index.md:44]: We also need to add the prerequisities for the apps that can be expedited in this manner:

To plug into this expedited flow, the desired app must meet the following requirements:
1. It must be a Store packaged app in MSIX format
2. It must have a valid Product Family Name (PFN)

File: desktop-src/updateorchestrator/index.md [hunk 2]
```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
...
+> Expediting Applications is only available for OEMs and only applicable in select regions. Expediting an application may have negative impacts to the out of box experience for new devices. This functionality is only available on select client builds.
+
+OEMs can register an update to be expedited by writing a new key in the existing staging folder for expedited apps: HKLM\Software\Microsoft\WindowsUpdate\Orchestrator\UScheduler_OOBE 
+
+Each OEM provider registration is in the form of a SubKey (with a unique name that will be used to identify this expedited app), and a set of registry values to indicate specific options for this update. The content of each key is in the following format: 
+
+Key name:  Unique name for this expedited app 
+
+Under Key name OEMs will need to create two additional keys:
+- updaterPriority: Number to indicate relative priority of this update
```
Comment (gerardoc-ms) [inline desktop-src/updateorchestrator/index.md:51]: Need to also provide information on the specific type of registry values to add:

Value name:  updaterPriority
Value type: REG_DWORD
Value data:  A numeric value from 1 - 100.  Lower values indicate a higher relative priority to other expedited apps.

Value name:  expeditedPayload
Value type:  REG_SZ
Value data:  A string representing a valid JSON blob with options for this specific updater

File: desktop-src/updateorchestrator/index.md [hunk 3]
```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
...
+
+Example: 
+{   "PFN": "PFNName", 
+    "Endpoint": “SSL_URI”, 
+    "AllowedInOobe": false, 
+    "MaxRetryCount": 3, 
+    “TimeoutDurationInMinutes”: 15, 
+    “ExcludedRegions”: [“CN”]   } 
+
+Overview of used terms:
```
Comment (gerardoc-ms) [inline desktop-src/updateorchestrator/index.md:62]: Rephrase to "Overview of contents of the JSON blob"

File: desktop-src/updateorchestrator/index.md [hunk 4]
```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
...
+    "Endpoint": “SSL_URI”, 
+    "AllowedInOobe": false, 
+    "MaxRetryCount": 3, 
+    “TimeoutDurationInMinutes”: 15, 
+    “ExcludedRegions”: [“CN”]   } 
+
+Overview of used terms: 
+| State  | Type | Key name | Description | 
+| ------ | ---- | -------- | ----------- |
+| Required | String | PFN | The Package Family Name of the app (ex MicrosoftTeams_example) |
```
Comment (gerardoc-ms) [inline desktop-src/updateorchestrator/index.md:65]: For the sample, can we use an actual valid PFN?  For example, the Store PFN?  Microsoft.WindowsStore_8wekyb3d8bbwe

File: desktop-src/updateorchestrator/index.md [hunk 5]
```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
...
+| Required | String | Endpoint | A string URIs pointing to a location hosting an MSIX package. Must be an SSL URI. | 
+| Optional | Boolean | AllowedInOobe | Whether this expedited app should run during user OOBE |
+| Optional | Number | MaxRetryCount | The number of times this updater is allowed to retry after failure |
+| Optional | Number | TimeoutDurationInMinutes | The duration to wait for this updater to complete work | 
+| Optional | Array (String) | ExcludedRegions | CN, US for regions where this flow should not be executed |
```
Comment (gerardoc-ms) [inline desktop-src/updateorchestrator/index.md:70]: The description for this should read:

A JSON array of strings for regions where this app should not be expedited.  Each entry in the array corresponds to the 2 letter ISO 3166-1 country code of the desired region.  For example, ["US"] would prevent this flow on devices where the region is United States.

**Review by GrantMeStrength — APPROVED** [review]

File: desktop-src/updateorchestrator/index.md (hunk 1 above)
Comment (dishahpatel) [inline desktop-src/updateorchestrator/index.md:44]: Are we using update and application interchangeably?

**Pull request merged**